composer.json

```diff
@@ -14,6 +14,10 @@
             "app/WebsiteKernel.php"
         ]
     },
+    "autoload-dev": {
+        "psr-4": { "Tests\\": "tests/" },
+        "files": [ "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
```

Comment:
Why do we need this?

Reply:
not sure why its in [symfony-standard](https://github.com/symfony/symfony-standard/blob/3.3/composer.json#L14) did try to use the dump function did work without problems in the tests. so will remove it.